src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationPreambleController.cs

```diff
@@ -599,7 +616,6 @@ public async Task<IActionResult> ChangeApplicationProviderRoute(Guid application
             return View("~/Views/Roatp/SelectApplicationRoute.cshtml", model);
         }
 
-        [HttpPost]
```

Comment:
As we discussed, Ravi, I'm not a fan of just losing a VERB on a controller method, i'd typically prefer to add a new method with the missing verb , but in the case of 'UpdateApplicationProviderRoute', it's doing an awful lot already and it would be somewhat clunky to wrap all that in a private method for the POST and the GET, so in this case, I think it's acceptable.

Reply:
I am interested in knowing why this is taken  off?